deployment/manifests/demo/demo_deployment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: demo-exome-results-browsers
+  labels:
+    service: demo-exome-results-browsers
+    environment: qa
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: demo-exome-results-browsers
+  template:
+    metadata:
+      labels:
+        name: demo-exome-results-browsers
+    spec:
+      containers:
+        - name: app
+          image: gcr.io/exac-gnomad/exome-results-browsers
```

Comment:
I think by default this pulls the `latest` image. If you need to pull a specific image tag with your IBD branch in it, just add `:your-tag-id` to the end of this (and sub the image repo URL for gnomadev, etc, if that's where the image is living). e.g. `gcr.io/exac-gnomad/exome-results-browsers:rg-ib-demo`